.gitattributes

```diff
@@ -1,9 +1,10 @@
 hack/verify-flags/known-flags.txt merge=union
 test/test_owners.csv merge=union
 
-**/zz_generated.*.go -diff
-**/types.generated.go -diff
-**/generated.pb.go -diff
-**/generated.proto -diff
-**/types_swagger_doc_generated.go -diff
-docs/api-reference/** -diff
+**/zz_generated.*.go -diff linguist-generated=true
+**/types.generated.go -diff linguist-generated=true
+**/generated.pb.go -diff linguist-generated=true
+**/generated.proto -diff linguist-generated=true
+**/types_swagger_doc_generated.go -diff linguist-generated=true
+docs/api-reference/** -diff linguist-generated=true
```

Comment:
I think this should be marked as documentation, rather than generated.
https://github.com/github/linguist#documentation

Reply:
or both generated and documentation

Reply:
I think we want to see the documentation changes in review.